docs/concepts/what-is-a-cloud-namespace-name.md

```diff
@@ -13,7 +13,7 @@ It cannot be changed after the Namespace is provisioned.
 
 Each Namespace Name must conform to the DNS naming rules:
 
-- A Namespace Name must contain at least 1 character and no more than 34 characters.
+- A Namespace Name must contain at least 2 characters and no more than 34 characters.
 - A Namespace Name must begin or end with a letter or number and can contain only letters, numbers, and the hyphen (-) character.
```

Comment:
cc @mastermanu right now we require it to begin with a letter (not a number).

Reply:
correct